File: aoc/aoc2022/day/Day.cs
public class Day
{
    public static void Run()
    {
        using (StreamReader file = new StreamReader("day/p.in"))
        {
            string ln;
            while ((ln = file.ReadLine()) != null)
            {
                Console.WriteLine(ln);
            }

            file.Close();
        }
    }
}

File: aoc/aoc2019/day/Day.fs
module Day
open System.IO
open System

type Day() =
    static member Run() =
        use file = new StreamReader("day/p.in")

        let rec readLines() =
            match file.ReadLine() with
            | null -> ()
            | ln ->
                Console.WriteLine(ln)
                readLines()

        readLines()

